src/blob_gc_job_test.cc

```diff
@@ -84,14 +86,23 @@ class BlobGCJobTest : public testing::Test {
     ASSERT_OK(db_->Flush(fopts));
   }
 
+  void CompactAll() {
+    auto opts = db_->GetOptions();
+    auto compact_opts = CompactRangeOptions();
+    compact_opts.change_level = true;
+    compact_opts.target_level = opts.num_levels - 1;
+    compact_opts.bottommost_level_compaction = BottommostLevelCompaction::kSkip;
+    ASSERT_OK(db_->CompactRange(compact_opts, nullptr, nullptr));
+  }
+
   void DestroyDB() {
     Status s __attribute__((__unused__)) = db_->Close();
     assert(s.ok());
     delete db_;
     db_ = nullptr;
   }
 
-  void RunGC() {
+  void RunGC(bool expected = false) {
```

Comment:
Seems a lot similiar with `TEST_StartGC`. Maybe remove and update that function?

Reply:
let's just keep that in this PR. IMO `TEST_StartGC` is still too similar to `BackgroundGC`, we can merge them together.